src/Exceptionless.Core/Extensions/CacheClientExtensions.cs

```diff
@@ -16,10 +16,10 @@ public static class CacheClientExtensions {
 
         if (count.HasValue) {
             var incrementedValue = await client.IncrementAsync(key, value, timeToLive).AnyContext();
-            return incrementedValue;
-            if (incrementedValue < 0) {
-                return await client.IncrementAsync(key, Math.Abs(incrementedValue), timeToLive).AnyContext();
-            }
+            if (incrementedValue >= 0)
+                return incrementedValue;
+
+            return await client.IncrementAsync(key, Math.Abs(incrementedValue), timeToLive).AnyContext();
```

Comment:
Don't love this. It feels hacky, but I guess it will do for now.

Reply:
I don't love it either, I think we can revert this change soon after any existing data is cleaned up.